InterGalacticCurrencyConvertor/src/main/java/FindPackageClasses.java

```diff
@@ -0,0 +1,83 @@
+package main.java;
+
+import java.util.ArrayList;
+import java.util.Objects;
+import java.io.File;
+import java.net.URL;
+import java.nio.CharBuffer;
+import java.nio.file.Path;
+import java.util.Enumeration;
+import java.io.IOException;
+
+
+/**
+ * Loads the classes in a package contained within this project that implement
+ * the given interface.
+ */
+public class FindPackageClasses {
```

Comment:
Try to just delete this class for this round - it just complicates the overall code.

Reply:
removed